manifests/install.pp

```diff
@@ -24,10 +24,7 @@
   }
 
   if $python::manage_python_package {
-    package { 'python':
-      ensure => $python::ensure,
-      name   => $python,
-    }
+    ensure_packages('python')
```

Comment:
The actual package name is in `$python`.  I guess this also break any `$python::ensure` which is not `present`…

Reply:
It's just better in general to use `ensure_packages()` to avoid duplicate declaration of resources. According to the [changelog](https://github.com/voxpupuli/puppet-python/blob/master/CHANGELOG.md#1710-2014-09-25) this was already added in version 1.7.10 but according to the installation manifests it is not actually implemented.